src/components/Slider.react.js

```diff
@@ -115,10 +115,40 @@ Slider.propTypes = {
     }),
 
     /**
-     * Value by which increments or decrements are made
+     * Value by which increments or decrements are made.
      */
     step: PropTypes.number,
 
+    /**
+     * If true, display an Input component whose value is synced with the Slider's value.
+     */
+    syncedInput: PropTypes.bool,
```

Comment:
Per the discussion @chriddyp and I had just now and summarized on Slack: let's convert all these new props to `snake_case`. In the short term this will make slider a little funny since it already has some `camelCase` props. That's OK, we'll get to it soon, and adding the backward-compatible conversion of other props is going to take some more work, especially since we use `omit` in this component.